README.md

```diff
@@ -310,6 +325,54 @@ function assignId (req, res, next) {
 }
 ```
 
+#### arguments to custom token formats
+
+Example of a custom token format that uses an argument. It will output the
```

Comment:
How do you use this feature in your production systems? Just take that use-case (which is clearly real-world) and place it as the example?